service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/DocmosisParameterGenerator.java

```diff
@@ -2,14 +2,21 @@
 
 import uk.gov.hmcts.reform.fpl.enums.DocmosisTemplates;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
 import uk.gov.hmcts.reform.fpl.model.order.Order;
 import uk.gov.hmcts.reform.fpl.service.orders.docmosis.DocmosisParameters;
 
+import java.util.List;
+
 public interface DocmosisParameterGenerator {
 
     Order accept();
 
     DocmosisParameters generate(CaseData caseData);
 
     DocmosisTemplates template();
+
+    default List<DocumentReference> additionalDocuments(CaseData caseData) {
```

Comment:
I don't feel like this belongs as part of this interface. This is nothing to do with the DocmosisParameters. Instead it should probs be a new interface that is added to the OrderDocumentGeneratorHolder